src/utils/priceCalculating.js

```diff
@@ -1 +1,3 @@
 export const calcPriceForCart = (item, currency) => item.price[currency].value * item.quantity;
+export const min = (minPrice,currency) => parseInt(minPrice ? minPrice[currency].value / 100 : 0, 10);
```

Comment:
You should make a new fucntion, like:

const roundPrice = (price) => parseInt(price/100, 10)
And use it in min/max functions and not duplicate the code.